tests/scripts/pulp_rpm/test_rpm_sync_publish.sh

```diff
@@ -33,6 +33,11 @@ then
 else
   expect_succ pulp rpm repository sync --name "cli_test_rpm_repository" --skip-type srpm
 fi
+if pulp debug has-plugin --name "rpm" --min-version "3.22.0"
```

Comment:
Is RPM still following pulpcore versioning?

Reply:
Heh - no, that's just me being confused. Will fix.